README.md

```diff
@@ -1,3 +1,30 @@
-## PLDS
+# PLDS - Phoenix LiveDashboard Standalone
 
-Work in progress. :)
+PLDS is a command line interface for Phoenix LiveDashboard with some additional extensions.
+
+It's useful in environments where you can't install Phoenix or the dashboard.
+This can be used to access remote system that are reachable locally.
+
+## Usage
+
+You can install PLDS using the `mix escript.install hex plds` (or if it's not
+published yet, `mix escript.install github phoenixframework/plds`).
+
+Make sure the path used for installing escripts are available in your
+`PATH` variable.
+
+After that, you can start PLDS with the following command:
+
+    $ plds server --connect node_a,node_b --port 9000 --open
```

Comment:
Instead of comma, you can allow the --connect option to be given multiple times by using `:keep` as the type in OptionParser.